src/ui/popup/components/header/index.tsx

```diff
@@ -7,6 +7,7 @@ import SystemIcon from '../../main-page/system-icon';
 import WatchIcon from '../../main-page/watch-icon';
 import SiteToggle from '../site-toggle';
 import MoreToggleSettings from './more-toggle-settings';
+import Link from '../link';
```

Comment:
Remove Link component, refactoring should not be a part of this PR.

Reply:
This is not pure refactoring, it fixes the Thunderbird problem where `<a href="">` does not work (as described above). 
Otherwise the onclick arrow function with chrome.tabs API would have to be added everywhere, which seems like a lot of code duplication. Or what do you suggest instead?

Reply:
Same problem in Firefox, have a look at index.tsx file, it should have a custom click event handler

Reply:
On normal Firefox the anchor tags work as well. You are referring to `onMoreToggleSettingsClick`, that gets passed with the body state to the header?
So should a helper function be defined in `body.tsx` and passed as parameter to the header, instead of the component import?

Reply:
Try adding Thunderbird here https://github.com/darkreader/darkreader/blob/df8273af4462b27b59c802b4a1fbd77f536a7f99/src/ui/popup/index.tsx#L44

Reply:
Oh thanks, I didn't see that. 
It already uses all Firefox fixes though: `isFirefox = userAgent.includes('firefox') || isThunderbird;`. 
It just did not work, because https://github.com/darkreader/darkreader/blob/df8273af4462b27b59c802b4a1fbd77f536a7f99/src/ui/popup/utils/issues.ts#L40 immediately closes the tab again... On Thunderbird the popup closes anyway when switching tabs, so this is not not needed, I have added the isThunderbird check now there and removed the Link component.
Is this okay now?